app/helpers/spotlight/main_app_helpers.rb

```diff
@@ -27,4 +27,18 @@ def field_enabled? field, *args
     field.enabled && field.send((:show if controller.is_a? Blacklight::Catalog and ["edit", "show"].include?(action_name)) || document_index_view_type)
   end
 
+  def link_back_to_catalog(opts={:label=>nil})
+    opts[:route_set] ||= spotlight if current_exhibit
+    super
+  end
+
+  def render_save_search
```

Comment:
Yeah, off-hand, I don't understand why `app/views/catalog/_sort_and_per_page.html.erb` isn't `app/views/spotlight/catalog/_sort_and_per_page.html.erb`.. It seemed to work as expected after moving it.